README.md

```diff
@@ -5,17 +5,30 @@ This action uses [__flake8__](http://flake8.pycqa.org) to know if your Python co
 
 If syntax errors are found then this Action uses [__futurize__](http://python-future.org/futurize_cheatsheet.html) create pull requests that gradually upgrade that code to be more compatible with Python 3.  After this Action has run, return to your repo and look for a "__modernize-Python-2-codes__" branch in your repo.  If this branch exists then select it and __make pull request__ and make sure that your automate tests pass before merging the pull request.  After merging, delete the "__modernize-Python-2-codes__" branch so that the process can be repeated.
 
-Example workflow (Put the following text into your repo's `.github/main.workflow`):
+Example workflow (Put the following text into your repo's `.github/workflows/py3.yml`):
```

Comment:
Let's call it `upgrade_to_Python3.yml` instead of `py3.yml` to make its objective clear.